src/utils/lodashTemplate.ts

```diff
@@ -0,0 +1,51 @@
+import StyleDictionary from "style-dictionary";
+import _ from "lodash";
+import * as fs from "fs";
+import * as path from "path";
+import { FormatterArguments } from "style-dictionary/types/Format";
+
+function androidLicenseHeader() {
+    return `/*
+   * Copyright (c) __YEAR__ New Vector Ltd
+   *
+   * Licensed under the Apache License, Version 2.0 (the "License");
+   * you may not use this file except in compliance with the License.
+   * You may obtain a copy of the License at
+   *
+   *     http://www.apache.org/licenses/LICENSE-2.0
+   *
+   * Unless required by applicable law or agreed to in writing, software
+   * distributed under the License is distributed on an "AS IS" BASIS,
+   * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+   * See the License for the specific language governing permissions and
+   * limitations under the License.
+   */`.replace('__YEAR__', new Date().getFullYear().toString())
+  }
+  
+  export default function lodashTemplate(templatePath: string, args: FormatterArguments) {
```

Comment:
This function is too specialised for the android context. We can either make it more generic to fit broader use cases, or change the naming to make it valid on this use case only

Reply:
@germain-gg can you take care of this one?

Reply:
Actually, I think I found a way to achieve this in https://github.com/vector-im/compound-design-tokens/pull/48/commits/21fd90d3b675b2a91de6a5a026d7598e89cd1fe9.